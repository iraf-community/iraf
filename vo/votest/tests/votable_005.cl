#
#  Print VOTable column statistics

# Set the test description string.
votest.descr = "TABLES support: Print VOTable column statistics"

# Convert the data$logical to a local path.
s1 = data_url // "/usno-b.xml"
s2 = "file://" // data_path // "/usno-b.xml"
s3 = "file:///localhost" // data_path // "/usno-b.xml"


fcache init

# Execute the test commands.
print ("\nLogical Path:  data$usno-b.xml")		# logical path
tstat ("data$usno-b.xml", "B1")

print ("\nHTTP URI:  " // s1)				# remote http URI
tstat (s1, "B1")

print ("\nFile URI:  " // s2)				# file URI
tstat (s2, "B1")

print ("\nFile URI:  " // s3)				# file URI
tstat (s3, "B1")
