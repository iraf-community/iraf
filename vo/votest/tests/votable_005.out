
Logical Path:  data$usno-b.xml
# data$usno-b.xml  B1
# nrows            mean        stddev        median           min           max
   65       6.770615385       9.58356            0.            0.         20.84

HTTP URI:  http://iraf.noao.edu/votest//usno-b.xml
# http://iraf.noao.edu/votest//usno-b.xml  B1
# nrows            mean        stddev        median           min           max
   65       6.770615385       9.58356            0.            0.         20.84

File URI:  file:///iraf/iraf/vo/votest/data//usno-b.xml
# file:///iraf/iraf/vo/votest/data//usno-b.xml  B1
# nrows            mean        stddev        median           min           max
   65       6.770615385       9.58356            0.            0.         20.84

File URI:  file:///localhost/iraf/iraf/vo/votest/data//usno-b.xml
# file:///localhost/iraf/iraf/vo/votest/data//usno-b.xml  B1
# nrows            mean        stddev        median           min           max
   65       6.770615385       9.58356            0.            0.         20.84
